### PR TITLE
TASK06: memory allocation timings

### PR DESCRIPTION
**Pull request by lavrinovu4**

### CODE REVIEW

File: 06_memory/module/dump.log [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+[  298.835101]      8388608    0.002206364        0.001220781
+[  298.842035]     16777216    0.004284895        0.002591941
+[  298.855543]     33554432    0.008409522        0.004976955
+[  298.884494]     67108864    0.017163034        0.011618808
+[  298.884680]
+[  298.884680] ***** GET_FREE_PAGES *****
+[  298.884732]  Buffer size  allocation time  freeing time
+[  298.884902]         4096    0.000098796        0.000021249
+[  298.884941]         8192    0.000003362        0.000001051
+[  298.884981]        12288    0.000002121        0.000001024
```
Comment (ekovalyov) [inline 06_memory/module/dump.log:84]: Is it possible to allocate exactly 12288 bytes using get_free_pages?

File: 06_memory/module/module_malloc.c [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+	t[1] = ktime_get();
+	call_free(alloc, p, i);
+	t[2] = ktime_get();
+
+	t_a = ktime_to_timespec64(ktime_sub(t[1], t[0]));
+	t_f = ktime_to_timespec64(ktime_sub(t[2], t[1]));
+
+	if (alloc == GET_FREE_PAGES)
+		buffer_allocated = i * PAGE_SIZE;
```
Comment (ekovalyov) [inline 06_memory/module/module_malloc.c:77]: Is this conversation correct?

**Review by ekovalyov — CHANGES_REQUESTED** [review]
Please fix the issues in code.

File: 06_memory/module/module_malloc.c (hunk 2 above)
Comment (lavrinovu4) [reply 06_memory/module/module_malloc.c]: Agree, fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/148/commits/7181b5d0c66692b76dc01a72d39a7497a761e93c

File: 06_memory/module/dump.log (hunk 1 above)
Comment (lavrinovu4) [reply 06_memory/module/dump.log]: no, wrong formula was used. Fixed in https://github.com/Kernel-GL-HRK/gl_kernel_procamp_2021/pull/148/commits/7181b5d0c66692b76dc01a72d39a7497a761e93c

**Review by ekovalyov — APPROVED** [review]

**Pull request merged**